hbase-server/src/main/java/org/apache/hadoop/hbase/mob/DefaultMobStoreFlusher.java

```diff
@@ -280,7 +283,11 @@ protected void finalizeWriter(StoreFileWriter writer, long cacheFlushSeqNum,
     // The hfile is current up to and including cacheFlushSeqNum.
     status.setStatus("Flushing " + store + ": appending metadata");
     writer.appendMetadata(cacheFlushSeqNum, false);
-    writer.appendMobMetadata(mobRefSet.get());
+    //writer.appendMobMetadata(ImmutableSetMultimap.builder<TableName, String>().putAll(store.getTableName(),
+    //    mobRefSet.get()).build());
```

Comment:
same as above?

Reply:
sorry,  I can't seem to retrace the context for this comment. could you take a look at the most recent change set and see if you have a concern still?

Reply:
It was just about the 2 commented lines, unless there is something else to do, those shouldn't be there.

Reply:
ah. yep I took care of making sure no commented out lines in the checkstyle/javadoc commit.